src/main/java/sh/platform/config/reader/PlatformShException.java

```diff
@@ -1,11 +1,35 @@
 package sh.platform.config.reader;
 
+/**
+ * The root exceptions from the Platform.SH config reader.
+ */
 public class PlatformShException extends RuntimeException {
-    public PlatformShException(String message) {
+
+    /** Constructs a new runtime exception with the specified detail message.
+     * The cause is not initialized, and may subsequently be initialized by a
+     * call to {@link #initCause}.
+     *
+     * @param   message   the detail message. The detail message is saved for
+     *          later retrieval by the {@link #getMessage()} method.
+     */
+    PlatformShException(String message) {
         super(message);
     }
 
-    public PlatformShException(String message, Throwable cause) {
+    /**
```

Comment:
For my own education, why remove the public keyword?  At least in PHP I know it's very bad form to not specify visibility, ever.  Is it the opposite in Java?

Reply:
Other constructors in this library have a visibility set.

Reply:
Yes, fixed. Thank you.